src/test/scala/chiselTests/StrongEnum.scala

```diff
@@ -514,6 +515,23 @@ class StrongEnumSpec extends ChiselFlatSpec with Utils {
   it should "correctly check if the enumeration is one of the values in a given sequence" in {
     assertTesterPasses(new IsOneOfTester)
   }
+
+  it should "suppress warning using suppressEnumCastWarning" in {
```

Comment:
Could you also add a test where one warning is suppressed and the other is not. Then you can check that exactly one warning is generated.